iac-extensions/arm/src/test/java/org/sonar/iac/arm/tree/impl/bicep/PropertyImplTest.java

```diff
@@ -67,16 +81,23 @@ void shouldParseSimpleProperty() {
   }
 
   @Test
-  void shouldParseProperty() {
-    Assertions.assertThat(BicepLexicalGrammar.PROPERTY)
-      .matches("key:value")
-      .matches("key: value")
-      .matches("key :value")
-      .matches("key : value")
-      .matches("key1: value1")
-      .matches("Ke1: VALu3")
+  void shouldParsePropertyInterpString() {
+    String code = code("'key':value");
 
-      .notMatches("1key: 1value")
-      .notMatches("@abc x value");
+    Property tree = parse(code, BicepLexicalGrammar.PROPERTY);
+    assertThat(((StringLiteral) tree.value()).value()).isEqualTo("value");
+    assertThat(tree.is(ArmTree.Kind.PROPERTY)).isTrue();
+
+    InterpolatedString key = (InterpolatedString) tree.children().get(0);
```

Comment:
I think it's implicit that this is of kind `InterpolatedString `because it would crash otherwise, but I would add an assert 
 asserting it's Kind before.

Reply:
The others could benefit from it aswell IMO.